fmriprep/workflows/anatomical.py

```diff
@@ -45,26 +45,26 @@ def t1w_preprocessing(settings, name='t1w_preprocessing'):
                 'fs_2_t1_transform']), name='outputnode')
 
     # 0. Align and merge if several T1w images are provided
-    t1wmrg = pe.Node(IntraModalMerge(), name='MergeT1s')
+    t1wmrg = pe.Node(IntraModalMerge(), name='t1wmrg')
 
     # 1. Reorient T1
     arw = pe.Node(niu.Function(input_names=['in_file'],
                                output_names=['out_file'],
                                function=reorient),
-                  name='Reorient')
+                  name='arw')
 
     # 2. T1 Bias Field Correction
     # Bias field correction is handled in skull strip workflows.
 
     # 3. Skull-stripping
-    asw = skullstrip_wf()
+    asw = skullstrip_wf(name='asw')
     if settings.get('skull_strip_ants', False):
-        asw = skullstrip_ants(settings=settings)
+        asw = skullstrip_ants(name='asw', settings=settings)
```

Comment:
Giving `skullstrip_wf` and `skullstrip_ants` the name `'asw'` means if I ever re-run with/without ANTs, the `asw` directory is going to get clobbered. Perhaps in these swappable workflows, we could be slightly less rigid without losing anything?

```Python
skullstrip = skullstrip_wf(name='skullstrip_wf')
if settings.get('skull_strip_ants', False):
    skullstrip = skullstrip_ants(name='skullstrip_ants', settings=settings)
```

We keep the same prefix as the variable, the workflow name matches the workflow-generating function (which is where we'll want to look, if we're matching directories with variables) and there's no conflation of the two workflows in the working directory.